Tests/AppTests/Mocks/AppEnvironment+mock.swift

```diff
@@ -32,7 +32,7 @@ extension AppEnvironment {
             collectionSigningCertificateChain: {
                 [
                     SignedCollection.certsDir
-                        .appendingPathComponent("package_collections_dev.cer"),
+                        .appendingPathComponent("package_collections.cer"),
```

Comment:
This could actually be

```swift
            collectionSigningCertificateChain: AppEnvironment.live.collectionSigningCertificateChain,
            collectionSigningPrivateKey: AppEnvironment.live.collectionSigningPrivateKey,
```

Reply:
Amended 👍 but I'll wait to push as I want to see if this test run succeeds in CI.